docs/guides/sdk/javascript-typescript-sdk.mdx

```diff
@@ -211,3 +211,45 @@ if (last.role === MessageRole.assistant && last.toolCalls.length > 0) {
   ])
 }
 ```
+
+## Evaluating Conversations
+
+To evaluate a conversation using configured evaluators, use the `eval` method:
+
+```typescript
+import { Latitude } from '@latitude-data/sdk'
+
+// Initialize SDK
+const sdk = new Latitude(process.env.LATITUDE_API_KEY, {
+  projectId: 1,
+})
+
+// 1. Run the document
+const { uuid } = await sdk.run('joker', {
+  parameters: {
+    topic: 'police',
+  },
+  stream: false,
+})
+
+// 2. Chat with the model
+await sdk.chat(uuid, [
+  {
+    role: 'user',
+    content: [
+      {
+        type: 'text',
+        text: 'Tell me another joke about doctors',
+      },
+    ],
+  },
+])
+
+// 3. Evaluate the full conversation
+await sdk.eval(uuid)
+```
```

Comment:
Does this evaluate the log for ALL evaluations connected to the document?

Reply:
I love the caps in "ALL" for an extra those of drama. Yes it does but you can pass it an optional array of evaluation uuids to filter by the evaluations that you are interested in if you only wanna run a subset

Reply:
updated the docs